tedana/metrics/kundu_fit.py

```diff
@@ -116,16 +116,15 @@ def dependence_metrics(catd, tsoc, mmix, t2s, tes, ref_img,
     # correct mmix & WTS signs based on spatial distribution tails
     signs = stats.skew(WTS, axis=0)
     signs /= np.abs(signs)
-    mmix = mmix.copy()
-    mmix *= signs
+    mmix_new = mmix * signs
```

Comment:
I'd call it like this `mmix_signed = mmix * signs`. _New_ is not very explanatory imo.

Reply:
`mmix_new` is used later on for the sorted version, so this is just pushing that variable definition up a bit.

I'm happy to change it to something other than `mmix_new`, but I'd prefer that it be something that covers both the sign changes and the sorting, so that we don't end up with extraneous variables.

Reply:
Good point. What about `mmix_corrected`? It sounds reasonable to me given that the comment just above states we're correcting `mmix`.

Reply:
Sounds good. I'll push that now.